vep_mapping_pipeline/consequence_mapping.py

```diff
@@ -1,10 +1,6 @@
 #!/usr/bin/env python3
 
-"""
-Pipeline for mapping variants to the genes they affect and their functional consequences, using Ensembl VEP API.
-For documentation, refer to /README.md
-"""
```

Comment:
I can see that this docstring has been moved to the argparse description. Is it possible to have it in both places (share a common string)?

Reply:
Yes, turns out it is possible to simply reference the docstring via `__doc__`! Addressed in https://github.com/EBIvariation/vep-mapping-pipeline/pull/1/commits/742307d78921ca069d482303ac63273fb7a3a20a